src/PAO_grid_transform_module.f90

```diff
@@ -111,12 +114,14 @@ subroutine single_PAO_to_grid(pao_fns)
     use primary_module, ONLY: domain
     use cover_module, ONLY: DCS_parts
     use set_blipgrid_module, ONLY : naba_atoms_of_blocks
-    use angular_coeff_routines, ONLY : evaluate_pao
+    !use angular_coeff_routines, ONLY : evaluate_pao, pao_elem_derivative_2
     use functions_on_grid, ONLY: gridfunctions, fn_on_grid
     use pao_format
 
     implicit none
-    integer,intent(in) :: pao_fns
+    integer, intent(in) :: pao_fns
+    !procedure(evaluate_interface), intent(in) :: evaluate
+    integer, intent(in), optional :: direction
```

Comment:
I don't think that direction should be optional: it's always passed to evaluate.